src/halmos/__main__.py

```diff
@@ -422,120 +431,34 @@ def get_state_id(ex: Exec) -> bytes:
     return snapshot_state(ex, include_path=True).unwrap()
 
 
-def run_invariant_tests(
-    ctx: ContractContext, pre_ex: Exec, funsigs: list[str]
-) -> list[TestResult]:
+def compute_frontier(ctx: ContractContext, depth: int) -> Iterator[Exec]:
```

Comment:
I would name this `_compute_frontier` to signal that this is an "internal" kind of function and make a note that this is only to be called by `get_frontier`.

Reply:
https://github.com/a16z/halmos/pull/472/commits/7b9503493e36795034fd37cfce6423dfbd9bd7ba